packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -11,6 +11,8 @@ import 'io.dart';
 import 'terminal.dart';
 import 'utils.dart';
 
+const int kDefaultStatusPadding = 59;
```

Comment:
could you add 7 to lib/src/ios/mac.dart's lines 369 and 396? (we can probably make them relative vs this constant now)

Reply:
done